tag/nitag.yml

```diff
@@ -0,0 +1,1749 @@
+      description: Returns the current values of the tags in a selection, excluding timestamps
+      operationId: GetSelectionCurrentValues
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            type: array
+            items:
+              description: Tag path with value
+              type: object
+              title: Tag Path With Value
+              properties:
+                path:
+                  type: string
+                  description: Tag path
+                current:
+                  $ref: '#/definitions/TagValue'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections/{id}/values/current/timestamp:
+    get:
+      tags:
+        - selections
+      summary: Get the timestamps of the current values of the tags in a selection
+      description: Gets the timestamps of the current values of the tags in a selection
+      operationId: GetSelectionValuesCurrentTimestamp
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Array containing the paths of the tags in the selection, along with the timestamp of their current values
+            type: array
+            title: Tag Paths With Timestamps
+            items:
+              type: object
+              title: Tag Path and Timestamp
+              description: A tag path and the timestamp of its current value
+              properties:
+                timestamp:
+                  description: Timestamp associated with the current value
+                  type: string
+                  format: iso-date-time
+                  example: '2018-09-04T18:45:08Z'
+                path:
+                  type: string
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v2/selections/{id}/values/aggregates/min:
+    get:
+      tags:
+        - selections
+      summary: Get the "min" aggregate values for the tags in a selection
+      description: Returns the minimum aggregate values for the tags in a selection
+      operationId: GetSelectionMinValues
+      parameters:
+        - in: path
+          name: id
+          description: Selection ID
+          type: string
+          required: true
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Array of the "min" values from the aggregates for each tag in the selection.
+            type: array
+            items:
+              type: object
+              title: Tag Paths and Count Values
```

Comment:
min value.

Reply:
Went with "Tag Path with Min Aggregate" for consistency with the related routes.